openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -104,7 +104,7 @@ def create_account_with_params(request, params):
     # Copy params so we can modify it; we can't just do dict(params) because if
     # params is request.POST, that results in a dict containing lists of values
     params = dict(params.items())
-
+    req_via_social_auth = True if params.has_key('social_auth_provider') else False
```

Comment:
you can simply write `req_via_social_auth = params.has_key('social_auth_provider')` or `'social_auth_provider' in params`

Reply:
done